### ISSUE TITLE
missing digits on song timer

### ISSUE THREAD

**Issue by olldray** [mpogue2/SquareDesk#343]
The song timer on the right hand side of the song's progress bar shows "0:0  / 3:4"
It seems to be missing one seconds digit for both current and total time. I suspect that this is actually a font sizing issue, as there are other areas of the UI where it looks like the last letter of a word is cut off. Interestingly, changing the application font size (via ctrl+'+' or ctrl+'-') does not significantly alter the situation. The UI seems to expand/contract to fit the changing font sizes, always giving too little space to fit all the digits. Perhaps my system is lying about the size of the characters?

**Comment by mpogue2** [comment]
Yep -- that's probably a font size problem.  Font size is not portable across platforms in Qt, so it's tuned for each individual platform.
